# HG changeset patch
# User Thorsten Meyer <thorsten.meyier@gmx.de>
# Date 1264884719 -3600
#      Sat Jan 30 21:51:59 2010 +0100
# Node ID f63ee6a20f57ce5b32a0d91f1bbce0ed8b11471f
# Parent  0a5a769b8fc0f1adead54c2324c43ff1a2b5d578
Replace removed functions in tests

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2010-01-30  Thorsten Meyer  <thorsten.meyier@gmx.de>
+
+	* statistics/tests/kruskal_wallis_test.m: Replace chisquare_cdf
+	by chi2cdf.
+
 2010-01-29  Ben Abbott  <bpabbott@mac.com>
 
 	* plot/axis.m, plot/__go_draw_axes__.m, plot/colorbar.m,
 	plot/isocolors.m, plot/isonormals.m, plot/isosurface.m,
 	plot/plotyy.m, plot/polar.m, plot/surfl.m, 
 	plot/private/__actual_axis_position.m:
 	Replace dataaspectratio and dataaspectratiomode properties with
 	plotboxaspectratio and plotboxaspectratio properties.
diff --git a/scripts/statistics/tests/kruskal_wallis_test.m b/scripts/statistics/tests/kruskal_wallis_test.m
--- a/scripts/statistics/tests/kruskal_wallis_test.m
+++ b/scripts/statistics/tests/kruskal_wallis_test.m
@@ -82,17 +82,17 @@ function [pval, k, df] = kruskal_wallis_
   n = length (p);
   k = 12 * k / (n * (n + 1)) - 3 * (n + 1);
 
   ## Adjust the result to takes ties into account.
   sum_ties = sum (polyval ([1, 0, -1, 0], runlength (sort (p))));
   k = k / (1 - sum_ties / (n^3 - n));
 
   df = m - 1;
-  pval = 1 - chisquare_cdf (k, df);
+  pval = 1 - chi2cdf (k, df);
 
   if (nargout == 0)
     printf ("pval: %g\n", pval);
   endif
 
 endfunction
 
 ## Test with ties
diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2010-01-30  Thorsten Meyer  <thorsten.meyier@gmx.de>
+
+	* DLD-FUNCTIONS/time.cc: Replace struct_contains by isfield.
+
 2010-01-30  Jaroslav Hajek  <highegg@gmail.com>
 
 	* pt-assign.h (tree_multi_assignment::etype): Remove.
 	(tree_multi_assignment::tree_multi_assignment): Update.
 	(tree_multi_assignment::op_type): Update.
 	* pt-assign.cc (tree_multi_assignment::tree_multi_assignment): Update.
 	(tree_multi_assignment::rvalue): Update.
 	(tree_multi_assignment::dup): Update.
diff --git a/src/DLD-FUNCTIONS/time.cc b/src/DLD-FUNCTIONS/time.cc
--- a/src/DLD-FUNCTIONS/time.cc
+++ b/src/DLD-FUNCTIONS/time.cc
@@ -144,26 +144,26 @@ gmtime (time ())\n\
   return retval;
 }
 
 /*
 
 %!test
 %! ts = gmtime (time ());
 %! assert((isstruct (ts)
-%! && struct_contains (ts, "usec")
-%! && struct_contains (ts, "year")
-%! && struct_contains (ts, "mon")
-%! && struct_contains (ts, "mday")
-%! && struct_contains (ts, "sec")
-%! && struct_contains (ts, "min")
-%! && struct_contains (ts, "wday")
-%! && struct_contains (ts, "hour")
-%! && struct_contains (ts, "isdst")
-%! && struct_contains (ts, "yday")));
+%! && isfield (ts, "usec")
+%! && isfield (ts, "year")
+%! && isfield (ts, "mon")
+%! && isfield (ts, "mday")
+%! && isfield (ts, "sec")
+%! && isfield (ts, "min")
+%! && isfield (ts, "wday")
+%! && isfield (ts, "hour")
+%! && isfield (ts, "isdst")
+%! && isfield (ts, "yday")));
 
 %!error <Invalid call to gmtime.*> gmtime ();
 
 %!error <Invalid call to gmtime.*> gmtime (1, 2);
 
 */
 
 DEFUN_DLD (localtime, args, ,
@@ -208,26 +208,26 @@ localtime (time ())\n\
   return retval;
 }
 
 /*
 
 %!test
 %! ts = localtime (time ());
 %! assert((isstruct (ts)
-%! && struct_contains (ts, "usec")
-%! && struct_contains (ts, "year")
-%! && struct_contains (ts, "mon")
-%! && struct_contains (ts, "mday")
-%! && struct_contains (ts, "sec")
-%! && struct_contains (ts, "min")
-%! && struct_contains (ts, "wday")
-%! && struct_contains (ts, "hour")
-%! && struct_contains (ts, "isdst")
-%! && struct_contains (ts, "yday")));
+%! && isfield (ts, "usec")
+%! && isfield (ts, "year")
+%! && isfield (ts, "mon")
+%! && isfield (ts, "mday")
+%! && isfield (ts, "sec")
+%! && isfield (ts, "min")
+%! && isfield (ts, "wday")
+%! && isfield (ts, "hour")
+%! && isfield (ts, "isdst")
+%! && isfield (ts, "yday")));
 
 %!error <Invalid call to localtime.*> localtime ();
 
 %!error <Invalid call to localtime.*> localtime (1, 2);
 
 */
 
 DEFUN_DLD (mktime, args, ,
